### DISCUSSION TITLE
Minimum System Config [Oct 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4151] [Discord Threads]
> 🙋‍♂️ **prasanth**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### NocoDB consumes roughly about 150 MB of RAM. So a 256MB machine should be good enough to get started. And depending on your project complexity & number of users to be supported - you can scale up RAM  
#  
> 🙋‍♂️ **prasanth**  
#### thanks for the reply <@857998597474484295>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1027559397611810898)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
